tasks/docker.rb

```diff
@@ -35,7 +31,8 @@ def install_ssh_components(distro, version, container)
                        'do sleep 0.1 ; count=$((count+1)) ; done ; fi'
       docker_exec(container, "bash -c '#{check_init_cmd}'")
     end
-    docker_exec(container, 'yum install -y sudo openssh-server openssh-clients')
+    packager = (version.to_i > 8) ? 'dnf' : 'yum'
```

Comment:
Isn't that off by one? EL8 already contains dnf?

Reply:
Could be, but I intentionally chose 9 as RedHat technically called it "yum 4" until EL 9.